test/services/user/services/AdminUsers.test.js

```diff
@@ -70,6 +70,50 @@ describe('AdminUsersService', () => {
 		expect(searchClass(result.data, '2A')).to.be.true;
 	});
 
+	it('only shows current classes', async () => {
+		const teacher = await testObjects.createTestUser({ roles: ['teacher'] });
+		const student = await testObjects.createTestUser({ firstName: 'Max', roles: ['student'] });
+		const currentSchool = await app.service('schools').get(teacher.schoolId);
+
+		const activeYear = currentSchool.years.activeYear._id;
+		const lastYear = currentSchool.years.lastYear._id;
```

Comment:
Like above, I think this might fail during the summer break (`activeYear` will be `null`).